### PR TITLE
Fix some lint issues.

### PR DESCRIPTION
**Pull request by jcdesimp**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,6 +1,13 @@
 {
     "extends": "airbnb",
      "rules": {
-        "no-console": 0
+        "no-console": 0,
+        "comma-dangle": ["error", {
```
Comment (chtinahow) [inline .eslintrc.json:5]: Is this getting rid of the trailing commas or what?

Comment (jcdesimp) [reply .eslintrc.json:5]: The original linter recommends trailing commas in arrow functions as well, which actually causes a syntax error (I think its only valid in ecma7, the backend however uses only ecma6. So I basically overrode the rule to make it so you DON'T do trailing commas in arrow functions. I kept the rest enabled tho cuz I do like the benefit of trailing commas in newline-separated lists and objects.

**Comment by chtinahow** [comment]
lgtm then! 👍

**Review by chtinahow — APPROVED** [review]
Looks good to me after Joe's comment! 👍

**Pull request merged**